### ISSUE TITLE
[Paper Note] DataFinder: Scientific Dataset Recommendation from Natural Language Descriptions, Vijay Viswanathan+, arXiv'23, 2023.05

### ISSUE THREAD

**Issue by AkihikoWatanabe** [AkihikoWatanabe/paper_notes#853]
# URL
- https://arxiv.org/abs/2305.16636
# Authors
  - Vijay Viswanathan
  - Luyu Gao
  - Tongshuang Wu
  - Pengfei Liu
  - Graham Neubig
# Abstract
  - Modern machine learning relies on datasets to develop and validate research ideas. Given the growth of publicly available data, finding the right dataset to use is increasingly difficult. Any research question imposes explicit and implicit constraints on how well a given dataset will enable researchers to answer this question, such as dataset size, modality, and domain. We operationalize the task of recommending datasets given a short natural language description of a research idea, to help people find relevant datasets for their needs. Dataset recommendation poses unique challenges as an information retrieval problem; datasets are hard to directly index for search and there are no corpora readily available for this task. To facilitate this task, we build the DataFinder Dataset which consists of a larger automatically-constructed training set (17.5K queries) and a smaller expert-annotated evaluation set (392 queries). Using this data, we compare various information retrieval algorithms on our test set and present a superior bi-encoder retriever for text-based dataset recommendation. This system, trained on the DataFinder Dataset, finds more relevant search results than existing third-party dataset search engines. To encourage progress on dataset recommendation, we release our dataset and models to the public.
# Translation (by gpt-5-nano)
- 現代の機械学習は研究アイデアの開発と検証のためにデータセットに依存している。公開データの増加を踏まえ、適切なデータセットを使うことはますます難しくなっている。どの研究課題にも、データセットのサイズ、モダリティ、ドメインなど、研究者がこの課題に答える能力をどれだけ引き出せるかに、明示的・暗黙的な制約が課される。短い自然言語による研究アイデアの説明を与えられたときにデータセットを推奨するタスクを実運用化して、必要に応じて関連するデータセットを見つけるのを助ける。データセット推奨は情報検索問題として独自の課題を呈する。データセットは検索の直接的なインデックスづけが難しく、このタスクに適したコーパスはすぐには入手できない。この課題を促進するために、より大きな自動構築の学習データセット（17,500件のクエリ）と、専門家が注釈を付けた評価データセット（392件のクエリ）から成るDataFinderデータセットを構築した。このデータを用いて、テストセット上でさまざまな情報検索アルゴリズムを比較し、テキストベースのデータセット推奨に対して優れたバイエンコーダー・リトリーバを提示する。DataFinderデータセットで訓練されたこのシステムは、既存のサードパーティのデータセット検索エンジンよりも、より関連性の高い検索結果を見つける。データセット推奨の進展を促すため、私たちはデータセットとモデルを公開する。
# Summary (by gpt-4o-mini)
- 短い自然言語の研究アイデアに基づいて、関連データセットを推奨するタスクを実運用化。データセット推奨は情報検索の問題として独自の課題を持つが、DataFinderデータセットを構築し、17,500件のクエリからなる自動学習データと392件の専門家注釈付き評価データを用いて検証。テスト結果からは、データセット推奨において高い関連性を示すバイエンコーダー・リトリーバが確認された。データセットとモデルは公開し、さらなる進展を促進。

**Comment by AkihikoWatanabe** [comment]
__translate: Modern machine learning relies on datasets to develop and validate research ideas. Given the growth of publicly available data, finding the right dataset to use is increasingly difficult. Any research question imposes explicit and implicit constraints on how well a given dataset will enable researchers to answer this question, such as dataset size, modality, and domain. We operationalize the task of recommending datasets given a short natural language description of a research idea, to help people find relevant datasets for their needs. Dataset recommendation poses unique challenges as an information retrieval problem; datasets are hard to directly index for search and there are no corpora readily available for this task. To facilitate this task, we build the DataFinder Dataset which consists of a larger automatically-constructed training set (17.5K queries) and a smaller expert-annotated evaluation set (392 queries). Using this data, we compare various information retrieval algorithms on our test set and present a superior bi-encoder retriever for text-based dataset recommendation. This system, trained on the DataFinder Dataset, finds more relevant search results than existing third-party dataset search engines. To encourage progress on dataset recommendation, we release our dataset and models to the public.
